onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -162,7 +169,10 @@ TEST(SplitOperatorTest, Axis0UnequalSplitFloat) {
                      {3.f, 4.f,
                       5.f, 6.f,
                       7.f, 8.f}});
-  // TensorRT parser: Assertion failed: axis != BATCH_DIM
+// TensorRT parser: Assertion failed: axis != BATCH_DIM
+#ifdef USE_COREML
+  RunTest<float>(axis, splits, input, outputs, {kTensorrtExecutionProvider}, false, true);
+#endif
```

Comment:
Do we need to limit this sort of testing to CoreML or is it valid to run these tests with `split` as input for all EPs and we can remove the `ifdef`s?

Reply:
removed.